scss/jquery/overrides/_select2.scss

```diff
@@ -0,0 +1,246 @@
+// Single select2
+.select2-container {
+  // The input field overlay
+  .select2-choice {
+    background: $crm-white;
+    border-color: $crm-grayblue-darker;
+    border-radius: $border-radius-base;
+    box-shadow: $crm-form-control-shadow;
+    box-sizing: border-box;
+    height: $input-height-base;
+    line-height: 27px;
+
+    &:focus {
+      outline: 0;
+    }
+
+    // The X icon to deselect the option
+    abbr {
+      background-image: none !important;
+      right: 30px;
+      top: 1px;
+
+      &::before {
+        @include fa-icon();
+        color: $brand-primary;
+        content: $fa-var-times-circle;
+        font-size: $font-size-base;
+      }
+    }
+
+    .select2-arrow {
+      background: $crm-white;
+      border-left-color: $crm-grayblue-darker;
+      border-radius: 0 $border-radius-base $border-radius-base 0;
+      text-align: center;
+      width: 26px;
```

Comment:
Why `26px`?

Reply:
No particular reason, is just how much the arrow needs to be wide according to the designs

Reply:
created a variable